understanding/22/visible-controls.html

```diff
@@ -40,29 +40,31 @@ <h2>Intent of Visible Controls</h2>
  
       <h3>Mobility and motor impairments</h3>
 
-      <p>People who use alternative input methods may have difficulty locating and operating controls that display only on hover or focus. For example, speech recognition users activate controls by speaking the name of the control. Functions that are only exposed through hover interaction or keyboard focus pose significant barriers to alternative input methods, such as speech.</p>
+      <p>People who use alternative input methods may have difficulty locating and operating controls that display only on hover or focus. For example, speech recognition users activate controls by speaking the name of the control. Functions that are only exposed through hover interaction or keyboard focus can pose significant barriers to alternative input methods.</p>
 
       <h3>When Needed</h3>
       
       <p>Information needed to identify controls does not need to be visible all the time. But information needed to identify controls must be visible <i>when the controls are needed</i> without hover interaction or keyboard focus. Controls can be available through a persistent <b><i>visible entry point</i></b>, such as a menu button that opens subitems. In a multi-step process or multi-part form, controls may be hidden in an earlier step or part; however, at the time the user can move the process forward, the information needed to identify the control, which is usually the control itself, must be visible without hover interaction or keyboard focus.</p>
 
-		<p>In some cases, controls are provided in multiple locations on a page or at multiple points within a process. In these cases, at least one of the instances of the control must be visible without hover interaction or keyboard focus. For example, on an email listing page some controls such as trash may be visible using pointer hover in the inbox list view. However, those controls are always visible on the email page. Because the controls are persistently visible on the email view, they do not need to be persistently visible on the inbox view.</p>
+		<p>In some cases, controls are provided in multiple locations on a page or at multiple points within a process. In these cases, at least one of the instances of the control must be visible without hover interaction or keyboard focus. For example, with an email reader some controls such as Trash may be visible when hovering over individual messages in the inbox list view. However, those controls will be always visible on the email view. Because the controls are persistently visible on the email view, they do not need to be persistently visible on the inbox view.</p>
```

Comment:
`will be always visible` sounds stilted. `will always be visible`?

Reply:
adjusted